guides/doc-Provisioning_Guide/topics/proc_installing-the-discovery-service.adoc

```diff
@@ -0,0 +1,38 @@
+[[installing-the-discovery-service]]
+= Installing the Discovery Service
+
+Complete the following procedure to enable the Discovery service on {SmartProxyServer}.
+
+.Procedure
+
+. Enter the following commands on {SmartProxyServer}:
++
+ifeval::["{build}" == "satellite"]
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foremain-maintain} packages install foreman-discovery-image rubygem-smart_proxy_discovery
+----
+endif::[]
+ifeval::["{build}" == "foreman"]
+[options="nowrap" subs="+quotes,attributes"]
+----
+# yum install foreman-discovery-image rubygem-smart_proxy_discovery tfm-rubygem-foreman_discovery
```

Comment:
For foreman there is no `foreman-discovery-image` package available. See above for the procedure, it's different. Only the FDI (RPM/ISO) artefact is different tho, all the rest is the same.